src/client.js

```diff
@@ -28,7 +28,9 @@ IrcClient.prototype._applyDefaultOptions = function(user_options) {
         nick: 'ircbot',
         username: 'ircbot',
         gecos: 'ircbot',
-        encoding: 'utf8'
+        encoding: 'utf8',
+        ping_interval: 30,
+        ping_timeout: 120,
```

Comment:
No, once a ping is sent, no more is sent until we get something from the server to reset it. The two timers shouldn't actually ever be active at the same time.